From 90ed12a7912f738dfb52267f58848f1ba49f1c02 Mon Sep 17 00:00:00 2001
From: Ruty Rinott <ruty@fb.com>
Date: Wed, 21 Aug 2019 10:39:02 -0700
Subject: [PATCH] removing gluster paths from OSS configs

Summary: D16708085 changed OSS config to point to gluster paths, reverting back to OSS paths

Reviewed By: borguz

Differential Revision: D16941400

fbshipit-source-id: 8054eb36d0c97eadd99e1ab561789ac123d84fd1
---
 demo/atis_joint_model/atis_joint_config.json | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/demo/atis_joint_model/atis_joint_config.json b/demo/atis_joint_model/atis_joint_config.json
index b96dd00a..772a6333 100644
--- a/demo/atis_joint_model/atis_joint_config.json
+++ b/demo/atis_joint_model/atis_joint_config.json
@@ -22,7 +22,7 @@
           },
           "word_embedding": {
             "embed_dim": 100,
-            "pretrained_embeddings_path": "/mnt/vol/gfsfblearner-carolina/admarket/ads_profile/glove/glove.6B.100d.txt",
+            "pretrained_embeddings_path": "demo/atis_joint_model/glove.6B.100d.txt",
             "embedding_init_strategy": "zero"
           },
           "output_layer": {
@@ -56,9 +56,9 @@
                   "doc_weight",
                   "word_weight"
                 ],
-                "train_filename": "/mnt/vol/pytext/users/arbabu/atis/atis.processed.train.csv_with_weights",
-                "eval_filename": "/mnt/vol/pytext/users/arbabu/atis/atis.processed.val.csv_with_weights",
-                "test_filename": "/mnt/vol/pytext/users/arbabu/atis/atis.processed.test.csv_with_weights"
+                "train_filename": "demo/atis_joint_model/atis.processed.train.csv",
+                "eval_filename": "demo/atis_joint_model/atis.processed.val.csv",
+                "test_filename": "demo/atis_joint_model/atis.processed.test.csv"
               }
             },
             "batcher": {
